src/main/scala/it/pagopa/interop/backendforfrontend/common/HeaderUtils.scala

```diff
@@ -0,0 +1,13 @@
+package it.pagopa.interop.backendforfrontend.common
+
+import it.pagopa.interop.commons.utils.AkkaUtils._
+
+import akka.http.scaladsl.model.HttpHeader
+import akka.http.scaladsl.model.headers.RawHeader
+import it.pagopa.interop.commons.utils.CONTENT_LANGUAGE
+
+object HeaderUtils {
+  def headersFromContext()(implicit contexts: Seq[(String, String)]): List[HttpHeader] = {
+    List(getAcceptLanguage(contexts).toOption.map(RawHeader(CONTENT_LANGUAGE, _))).flatten
+  }
```

Comment:
Minor readability improvement
```suggestion
  def headersFromContext()(implicit contexts: Seq[(String, String)]): List[HttpHeader] = 
    getAcceptLanguage(contexts).fold(_ => Nil, RawHeader(CONTENT_LANGUAGE, _))
  
```